presidio-analyzer/presidio_analyzer/recognizer_registry/recognizer_registry.py

```diff
@@ -198,6 +200,12 @@ def get_recognizers(
 
         return list(to_return)
 
+    def add_recognizer_map(self, all_entity_recognizers, supported_entity, rec):
```

Comment:
Can you please make this private + add type hints?

Reply:
Furthermore, there are a few linting issues as can be seen here: https://dev.azure.com/csedevil/Presidio/_build/results?buildId=16970&view=logs&jobId=248a81d7-74ff-5424-a6bb-5f1f442ace41&j=248a81d7-74ff-5424-a6bb-5f1f442ace41&t=b4a88934-08b9-5ac5-8cd9-057bdd5531eb

Consider using ruff to test